views/filter/editfilters.html

```diff
@@ -15,8 +15,8 @@
         </div>
       </div>
       <ul class="nav nav-tabs flex-shrink-0">
-        <li class="nav-item"><a class="nav-link" ng-class="{active: tab == 'basic'}" ng-click="filters.setTab('basic')">Basic</a></li>
-        <li class="nav-item"><a class="nav-link" ng-class="{active: tab == 'advanced'}" ng-click="filters.setTab('advanced')">Advanced</a></li>
+        <li class="nav-item"><a href class="nav-link" ng-class="{active: tab == 'basic'}" ng-click="filters.setTab('basic')">Basic</a></li>
```

Comment:
It sounds like the only thing preventing us from using `div` here is the lack of `cursor: pointer`. Is that accurate? If so, I wouldn't be opposed to adding that to `.nav-link` in our Bootstrap overrides. I agree with @wallw-bits on not using anchor tags for things that aren't links.